plugins/woocommerce/tests/e2e-pw/utils/plugin-utils.js

```diff
@@ -52,47 +54,101 @@ export const deletePlugin = async ( {
 	// If installed, get its `plugin` value and use it to deactivate and delete it.
 	if ( pluginToDelete ) {
 		const { plugin } = pluginToDelete;
+		const requestURL = `/wp-json/wp/v2/plugins/${ plugin }`;
+		let response;
 
-		await apiContext.put( `/wp-json/wp/v2/plugins/${ plugin }`, {
+		// Some plugins do not respond to some of the WP REST API endpoints like Contact Form 7.
+		// Print warning in such cases.
+		const warn = async ( response, warningMsg ) => {
+			console.warn( warningMsg );
+			console.warn(
+				`Response status: ${ response.status() } ${ response.statusText() }`
+			);
+			console.warn( `Response body:` );
+			console.warn( await response.json() );
+			console.warn( '\n' );
+		};
+
+		response = await apiContext.put( requestURL, {
```

Comment:
For some reason, Contact Form 7 cannot be deactivated and deleted through WordPress REST API 🤷 so I had this test print a warning instead. 

Plugin tests are run on localhost anyway, not on the daily smoke test site, so I think it's fine for now if Contact Form 7 wasn't deleted at the end of the test.